chat-plugins/scavengers.js

```diff
@@ -144,13 +197,86 @@ exports.commands = {
 		scavengers.answers = null;
 		scavengers.participants = {};
 		scavengers.finished = [];
+		scavengers.updatinglb = false;
+		scavengers.numblitzes = 0;
 		if (scavengers.result) {
 			Rooms('scavengers').addRaw('<div class="broadcast-blue"><strong>' + scavengers.result + '</strong></div>');
```

Comment:
I guess it's not an issue with your code, though, but if you want to fix that, that'd be cool.